winui/Avatar-view/how-to.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about using of the different states that exist in WinUi AvatarView control(SfAvatarView).
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# How to use different states
+
```

Comment:
@MadhumithanJayapathi How to use badge for WinUI AvatarView

Reply:
@Manivannan-E Changed the title as given.